<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1031</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="General">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>1010</width>
       <height>260</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Country:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Laboratory Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Laboratory Code:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>135</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Maximum 3 Characters</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>110</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Configuration File:</string>
      </property>
     </widget>
     <widget class="QLabel" name="general_configfile_name">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>110</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="general_human_radio">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>99</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Human</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="general_combined_radio">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Human, Animal, Food, Environment</string>
      </property>
     </widget>
     <widget class="QComboBox" name="general_country_combobox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>25</y>
        <width>511</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="general_labname_lineedit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>65</y>
        <width>511</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="general_labcode_lineedit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>105</y>
        <width>200</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="general_country_code">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>30</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Antibiotics">
    <attribute name="title">
     <string>Antibiotics</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>490</width>
       <height>331</height>
      </rect>
     </property>
     <property name="title">
      <string>Antibiotic list</string>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>35</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Guidelines: </string>
      </property>
     </widget>
     <widget class="QComboBox" name="antibiotic_guidelines_combobox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>371</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Test Method</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="antibiotics_disk_radio">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>99</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Disk</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="antibiotics_mic_radio">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>99</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>MIC</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="antibiotics_etest_radio">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>70</y>
        <width>99</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Etest</string>
      </property>
     </widget>
     <widget class="QListWidget" name="antibiotic_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>101</y>
        <width>471</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>298</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="antibiotics_search">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>295</y>
        <width>401</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="antibiotics_add_button">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>240</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="antibiotics_remove_button">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>320</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>120</y>
       <width>391</width>
       <height>331</height>
      </rect>
     </property>
     <property name="title">
      <string>Local Antibiotic List</string>
     </property>
     <widget class="QListWidget" name="antibiotic_local_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>371</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of antibiotics =</string>
      </property>
     </widget>
     <widget class="QLabel" name="num_antibiotics">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>300</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="antibiotics_full_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>495</width>
       <height>84</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>1. Choose the antibiotics which you test in your laboratory.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>2. Indicate the guidelines, the test method, and the antibiotics name.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>3. Print and review the antibiotic breakpoints.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>4. Define antibiotic panels (for data entry) and antibiotic profiles (for data analysis).</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>460</y>
       <width>971</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="antibiotics_breakpoint_button">
        <property name="text">
         <string>Breakpoints</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="antibiotics_expertrules_button">
        <property name="text">
         <string>Expert Rules</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="antibiotics_panels_button">
        <property name="text">
         <string>Panels</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="antibiotics_profiles_button">
        <property name="text">
         <string>Profiles</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="antibiotics_print_button">
        <property name="text">
         <string>Print</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Locations">
    <attribute name="title">
     <string>Locations - Optional</string>
    </attribute>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>841</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>The lists on the right will change the selected rows's value when clicked.</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="location_table_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>761</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>20</y>
       <width>221</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Institutions</string>
     </property>
     <widget class="QPushButton" name="location_institutions_edit_button">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>25</y>
        <width>50</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QListWidget" name="institution_listview">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>180</y>
       <width>221</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Departments</string>
     </property>
     <widget class="QPushButton" name="location_departments_edit_button">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>25</y>
        <width>50</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QListWidget" name="department_listview">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>340</y>
       <width>221</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>Location type</string>
     </property>
     <widget class="QListWidget" name="location_type_listview">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>201</width>
        <height>91</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="DataFields">
    <attribute name="title">
     <string>Data Fields - Optional</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>481</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string>Fields</string>
     </property>
     <widget class="QListWidget" name="data_fields_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>251</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="data_fields_modify_button">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>181</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Modify List</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number  of Fields =</string>
      </property>
     </widget>
     <widget class="QLabel" name="num_data_fields">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>320</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>90</y>
       <width>481</width>
       <height>351</height>
      </rect>
     </property>
     <property name="title">
      <string>Field Summary</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>91</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Length</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>111</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="data_fields_description">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="data_fields_name">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="data_fields_type">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="data_fields_len_spinbox">
         <property name="maximumSize">
          <size>
           <width>51</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>322</width>
       <height>62</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>Your data fields appear below.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_18">
        <property name="text">
         <string>Make any necessary changes.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>If you want to add or remove fields, select 'Modify list'.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="Alerts">
    <attribute name="title">
     <string>Alerts - Optional</string>
    </attribute>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>991</width>
       <height>241</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTableWidget" name="alerts_table_view">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>41</y>
        <width>971</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>671</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="alerts_new_button">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="alerts_edit_button">
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="alerts_delete_button">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="alerts_sort_combobox">
         <property name="minimumSize">
          <size>
           <width>301</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="alerts_group">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>991</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Organism</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="alerts_description">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>311</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Alert</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>30</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Active Rule</string>
      </property>
     </widget>
     <widget class="QListView" name="alerts_organism_listview">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Organisms</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Isolates</string>
      </property>
     </widget>
     <widget class="QListView" name="alerts_isolates_listview">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>391</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>70</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Alerts</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>130</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Messages</string>
      </property>
     </widget>
     <widget class="QListView" name="alerts_alerts_listview">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>70</y>
        <width>391</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QListView" name="alerts_messages_listview">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>130</y>
        <width>391</width>
        <height>71</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>30</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Priority</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>560</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="save_button">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Cancel </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
